### ISSUE TITLE
type attribute setting don't work on < jQuery 1.9

### ISSUE THREAD

**Issue by levkar** [CardJs/CardJs#8]
This expression make the library unusable with earlier versions of jQuery:

.attr("type", "tel");

Removal would be a good improvement for compatibility.

Thanks.

**Comment by pandymic** [comment]
Per the jQuery website, the `.attr()` method has always had support for a call with two string values, [as outlined here in the documentation](http://api.jquery.com/attr/#attr-attributeName-value).

The use of this mechanism should have no impact on jQuery all the way back to version 1.0

By nature, `.attr()` method simply inserts data in the the DOM with very little validation or error checking. It's up to the client browser to make due with the data that is set. If you're experiencing compatibility issues with legacy versions of jQuery, the root cause most likely lies elsewhere within the script.

**Comment by levkar** [comment]
You are right. But I needed this lines to commented out to get it working with jQuery 1.8.3

this.cvcInput.attr("type", "tel");

this.expiryMonthYearInput.attr("type", "tel");
